src/utils/colors.js

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+// https://github.com/FirefoxUX/design-tokens/tree/master/colors
+// Firefox Colors v1.0.3
```

Comment:
Does the npm package not give us what we need? I see a photon.json file in the github repo, would something like
```
import colors from 'json!photon-colors/photon.json';
export default colors;
```
work? Would need to `yarn add photon-colors` first, of course.

Reply:
> import colors from 'json!photon-colors/photon.json';

Probably not necessary to add `json!` because the json-loader is configured in webpack.